storm-server/src/main/java/org/apache/storm/scheduler/multitenant/Node.java

```diff
@@ -130,95 +133,110 @@ public static int countTotalSlotsAlive(Collection<Node> nodes) {
+        for (Set<WorkerSlot> slots : topIdToUsedSlots.values()) {
             total += slots.size();
         }
         return total;
     }
 
-    public int totalSlots() {
-        return totalSlotsFree() + totalSlotsUsed();
-    }
-
     public int totalSlotsUsed(String topId) {
         int total = 0;
-        Set<WorkerSlot> slots = _topIdToUsedSlots.get(topId);
+        Set<WorkerSlot> slots = topIdToUsedSlots.get(topId);
         if (slots != null) {
             total = slots.size();
         }
         return total;
     }
 
+    public int totalSlots() {
+        return totalSlotsFree() + totalSlotsUsed();
+    }
+
     private void validateSlot(WorkerSlot ws) {
-        if (!_nodeId.equals(ws.getNodeId())) {
-            throw new IllegalArgumentException(
-                "Trying to add a slot to the wrong node " + ws +
-                " is not a part of " + _nodeId);
+        if (!nodeId.equals(ws.getNodeId())) {
+            throw new IllegalArgumentException("Trying to add a slot to the wrong node "
+                    + ws
+                    + " is not a part of "
+                    + nodeId);
         }
     }
 
     private void addOrphanedSlot(WorkerSlot ws) {
-        if (_isAlive) {
-            throw new IllegalArgumentException("Orphaned Slots " +
-                                               "only are allowed on dead nodes.");
+        if (isAlive) {
+            throw new IllegalArgumentException("Orphaned Slots only are allowed on dead nodes.");
         }
         validateSlot(ws);
-        if (_freeSlots.contains(ws)) {
+        if (freeSlots.contains(ws)) {
             return;
         }
-        for (Set<WorkerSlot> used : _topIdToUsedSlots.values()) {
+        for (Set<WorkerSlot> used : topIdToUsedSlots.values()) {
             if (used.contains(ws)) {
                 return;
             }
         }
-        _freeSlots.add(ws);
+        freeSlots.add(ws);
     }
 
     boolean assignInternal(WorkerSlot ws, String topId, boolean dontThrow) {
         validateSlot(ws);
-        if (!_freeSlots.remove(ws)) {
-            for (Entry<String, Set<WorkerSlot>> topologySetEntry : _topIdToUsedSlots.entrySet()) {
+        if (!freeSlots.remove(ws)) {
+            for (Entry<String, Set<WorkerSlot>> topologySetEntry : topIdToUsedSlots.entrySet()) {
                 if (topologySetEntry.getValue().contains(ws)) {
                     if (dontThrow) {
-                        LOG.warn("Worker slot [" + ws + "] can't be assigned to " + topId +
-                                 ". Its already assigned to " + topologySetEntry.getKey() + ".");
+                        LOG.warn("Worker slot ["
+                                + ws
+                                + "] can't be assigned to "
+                                + topId
+                                + ". Its already assigned to "
+                                + topologySetEntry.getKey()
+                                + ".");
```

Comment:
This kind of enforced alignment will further encourage the use of format strings rather than string concatenation, because it is much less readable. I think this is a good thing.

I am not suggesting we change all such occurrences to use format strings now—if we see a performance issue or if we want to change it for readability later I consider it out of scope for this pull request.